website/docs/d/workspace_ids.html.markdown

```diff
@@ -22,17 +22,23 @@ data "tfe_workspace_ids" "all" {
   names        = ["*"]
   organization = "my-org-name"
 }
+
+data "tfe_workspace_ids" "prod-apps" {
+  tag_names         = ["prod", "app", "aws"]
+  organization = "my-org-name"
+}
 ```
 
 ## Argument Reference
 
-The following arguments are supported:
+The following arguments are supported. Either one of `names` or `tag_names` or both is required.
```

Comment:
Nit on the wording: `At least one of `names` or `tag_names` must be present. Both can be used together.`